### ISSUE TITLE
`sbt docker:stage` fails on Windows 10

### ISSUE THREAD

**Issue by oporkka** [sbt/sbt-native-packager#823]
This is related to #573 

Steps to reproduce on Windows 10 machine (I tried using both bash.exe and cmd.exe):

- Check out `sbt-native-packager` master branch
- Go to test-project-docker folder
- Run `sbt docker:stage`

You get stack trace

```
java.lang.ArrayIndexOutOfBoundsException: 1
        at com.typesafe.sbt.packager.docker.DockerPlugin$.com$typesafe$sbt$packager$docker$DockerPlugin$$makeAdd(DockerPlugin.scala:162)
        at com.typesafe.sbt.packager.docker.DockerPlugin$$anonfun$projectSettings$8.apply(DockerPlugin.scala:89)
        at com.typesafe.sbt.packager.docker.DockerPlugin$$anonfun$projectSettings$8.apply(DockerPlugin.scala:80)
        at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
        at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
        at sbt.std.Transform$$anon$4.work(System.scala:63)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
        at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
        at sbt.Execute.work(Execute.scala:237)
        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
        at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
        at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
        at java.util.concurrent.FutureTask.run(Unknown Source)
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
        at java.util.concurrent.FutureTask.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
```

What seems to have happened is that https://github.com/sbt/sbt-native-packager/commit/f52167977e74187a284f5b8392e3bdb64c05b4b6 (#573, #588) has been reverted by https://github.com/sbt/sbt-native-packager/commit/101311e4f967cc0117e05358960097b5915b2b38 (#760, #765)  in 1.1.1

**Comment by muuki88** [comment]
Thank for being patient with this windows/docker issue and for the research to fix this regression we introduced.

**Comment by morrobkg** [comment]
With the latest version (1.1.2), I'm getting this same error on a :stage & :publish.
Docker version: 1.10.0
Docker-machine version: 0.6.0
Windows 7
Scala 2.11.8

Originally had to update to latest for the -f deprecated fix. Are there any other changes since 1.1.0 I should be aware of?

**Comment by muuki88** [comment]
I think we haven't back-ported the fix to `1.1.x` yet. Would you mind making a pull request to the `1.1.x` branch with these changes: https://github.com/sbt/sbt-native-packager/pull/824

**Comment by oporkka** [comment]
@muuki88 added #842 for backport
